### ISSUE TITLE
Questions: About train BERT from scratch parameter?

### ISSUE THREAD

**Issue by HadXu** [songlab-cal/tape#46]
Hi, I want train a bert from scartch, But I have occured some problem. The first one is your parameter about `tape-train-distributed transformer masked_language_modeling --batch_size BS --learning_rate LR --fp16 --warmup_steps WS --nproc_per_node NGPU --gradient_accumulation_steps NSTEPS`, because when I train from scratch, the log follow

![image](https://user-images.githubusercontent.com/13307469/79623723-6c25ba00-8150-11ea-99d3-0b1e46575eb3.png)

and my parameter is

`DATA_DIR=data/
BS=1024
LR=0.0005
NGPU=8
GA=32

tape-train-distributed  transformer masked_language_modeling \
                                --data_dir ${DATA_DIR} \
                                --batch_size ${BS} \
                                --learning_rate ${LR} \
                                --warmup_steps 5000 \
                                --fp16 \
                                --nproc_per_node ${NGPU} \
                                --gradient_accumulation_steps ${GA}`

so I want konw your parameter？ 

thank you!

**Comment by rmrao** [comment]
The parameters used were:


    batch size: 512
    learning rate: 1e-3
    warmup steps: 10000
    epochs: 10

You can set nproc per node / gradient accumulation steps based on your environment (these do not affect training).
